contrib/win32/openssh/config.h.vs

```diff
@@ -207,13 +207,13 @@
 /* #undef HAVE_ADDR_V6_IN_UTMPX */
 
 /* Define to 1 if you have the `arc4random' function. */
-#define HAVE_ARC4RANDOM 1
```

Comment:
Why are these commented out?

Reply:
The symbols are no longer exported by LibreSSL, so we need to compile them at runtime in openbsd-compat.

Reply:
should we just remove them then?